test/quantum/__snapshots__/index-64.test.jsx.snap

```diff
@@ -357,6 +357,41 @@ exports[`renders correctly 1`] = `
           Speedometer
         </span>
       </h2>
+      <div
+        style={
+          Object {
+            "lineHeight": "8rem",
+            "textAlign": "center",
+            "width": "8rem",
+          }
+        }
+      >
+        <div
+          className="MuiCircularProgress-root-8 MuiCircularProgress-colorPrimary-11 MuiCircularProgress-indeterminate-10"
+          role="progressbar"
+          style={
+            Object {
+              "height": 40,
+              "width": 40,
+            }
+          }
+        >
+          <svg
+            className="MuiCircularProgress-svg-13"
+            viewBox="22 22 44 44"
+          >
+            <circle
+              className="MuiCircularProgress-circle-14 MuiCircularProgress-circleIndeterminate-16"
+              cx={44}
+              cy={44}
+              fill="none"
+              r={20.2}
+              strokeWidth={3.6}
+              style={Object {}}
+            />
+          </svg>
+        </div>
+      </div>
```

Comment:
Does this test now miss the point?  Showing just progress circles rather than content?

Reply:
I have not been able to get to the point where I know how to test things once there's data.
I need few days to study a good approach on how to add tests for chart.js 

For now, this only ensures that we're aware that things are changing.